apps/events/viewsets/invitation.py

```diff
@@ -0,0 +1,84 @@
+from drf_yasg.utils import swagger_auto_schema, no_body
+from rest_framework import viewsets, status
+from rest_framework.decorators import action
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from apps.events.models import Invitation, Event
+from apps.events.serializers import InvitationSerializer
+from apps.notifications.handlers.in_app import InAppNotificationsHandler
+from apps.notifications.managers.chain import NotificationsChainManager
+from apps.notifications.models import Notification
+from apps.profiles.utils import get_user_object, is_current_user
+
+
+class InvitationViewSet(viewsets.ModelViewSet):
+    model = Invitation
+    permission_classes = IsAuthenticated
+    queryset = Invitation.objects.all()
+    serializer_class = InvitationSerializer
+    http_method_names = ["get", "post", "delete"]
+
+    handlers = (InAppNotificationsHandler(),)
+    notifications_manager = NotificationsChainManager(handlers)
+
+    @swagger_auto_schema(request_body=no_body)
+    @action(methods=["post"], detail=True, url_name="invite_accept")
+    def invite_accept(self, request, event_id):
+        event = Event.objects.filter(event_id=event_id).first()
```

Comment:
This could be replaced with get(), which raises a 404 automatically if the object is not found
first() returns None, so you will need an explicit check and return a 404 if None is found